### DISCUSSION TITLE
The socket connections hung up after few days working with Brave browser

### DISCUSSION THREAD

**Discussion by mohammadranjbarz** [feathersjs/feathers#2370] [Q&A]
When we upgraded feathers and socketio this issue was solved but other issues come out 
https://github.com/feathersjs/feathers/issues/2337

So we had to downgrade socketio for solving #2337, but with doing that the Brave problems appear again,
I would appreciate it if you can help us.

PS : We work on an open-source project:
https://github.com/Giveth/giveth-dapp
https://github.com/Giveth/feathers-giveth
http://beta.giveth.io

**Comment by daffl** [comment]
Do you have any more info? I've never seen any issues with Brave and it should usually reconnect automatically. In general, Feathers doesn't really add much over Socket.io itself so it would be important to figure out more details on what is happening.
